build/publish.sh

```diff
@@ -3,78 +3,25 @@
 # Temp version
 VERSION="XXX"
 
-
-lint_and_test() {
-    echo "----------------------------------------------------"
-    echo "Running linter for version" $VERSION
-    echo "----------------------------------------------------"
-    if yarn run lint; then
-        echo "----------------------------------------------------"
-        echo "Done linting for version" $VERSION
-        echo "----------------------------------------------------"
-    else
-        echo "----------------------------------------------------"
-        echo "Failed linting!"
-        echo "----------------------------------------------------"
-        exit 1;
-    fi
-
-
-    echo "----------------------------------------------------"
-    echo "Running flow for version" $VERSION
-    echo "----------------------------------------------------"
-    if yarn run flow; then
-        echo "----------------------------------------------------"
-        echo "Done flowing for version" $VERSION
-        echo "----------------------------------------------------"
-    else
-        echo "----------------------------------------------------"
-        echo "Failed flowing!"
-        echo "----------------------------------------------------"
-        exit 1;
-    fi
-
-
-    echo "----------------------------------------------------"
-    echo "Running tests for version" $VERSION
-    echo "----------------------------------------------------"
-    if yarn run test; then
-        echo "----------------------------------------------------"
-        echo "Done testing for version" $VERSION
-        echo "----------------------------------------------------"
-    else
-        echo "----------------------------------------------------"
-        echo "Failed testing!"
-        echo "----------------------------------------------------"
-        exit 1;
+add_remote() {
```

Comment:
Consider combining these three script files to reuse these common utility methods. You could still maintain three separate "entry points" into the script.